src/sass/_common/header.scss

```diff
@@ -763,19 +779,43 @@ body .mobile {
 
         &-header {
             display: flex;
-            height: 40px;
+            width: 100%;
+            height: 50px;
             align-items: center;
+            line-height: 24px;
             border-bottom: 1px solid var(--general-section-1);
 
             > img {
                 border-right: 1px solid var(--general-section-1);
                 padding: 6px 8px;
+                padding: 6px 12px;
+            }
+            &-wrapper {
+                display: flex;
+                justify-content: space-between;
+                width: 100%;
             }
             &-text {
                 padding: 12px 16px;
                 font-size: $FONT_SIZE_N;
                 font-weight: 600;
             }
+            &-livechat {
+                align-items: center;
+                display: flex;
+                justify-content: flex-end;
+                padding-right: 1rem;
```

Comment:
Maybe we should stick to pixels because this project probably doesn't set `rem` size